app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -414,29 +414,14 @@ public void setCurrentSession(int aId) {
 
     public String setRegion(String aRegion) {
         Log.d(LOGTAG, "SessionStore setRegion: " + aRegion);
-        String result = "worldwide";
-        if (mCurrentSession == null || aRegion == null) {
-            return result;
-        }
-        State state = mSessions.get(mCurrentSession.hashCode());
-        if (state == null) {
-            return result;
-        }
-        result = state.mRegion = aRegion.toLowerCase();
-        return result;
+        aRegion = aRegion != null ? aRegion.toLowerCase() : "worldwide";
+        mRegion = aRegion;
```

Comment:
nit: maybe just save an assignment and assign directly to the field?

Reply:
yeah, probably should have - thanks